apps/weather_app/src/app/core/store/actions/city.actions.ts

```diff
@@ -0,0 +1,15 @@
+import { createAction, props } from '@ngrx/store';
+import { GeoCityData } from '../../models/cities.mode';
```

Comment:
- barrel import can be used
- absolute path with module ref can be used instead of long relative path